# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1393000536 18000
#      Fri Feb 21 11:35:36 2014 -0500
# Branch gui-release
# Node ID fb96b7f5524295c2e538106f6fe57746a8e12aba
# Parent  b75adad91fc62e6356294b4e4b281d6ad3643fe4
rename file to avoid clash on case-insenstive filesystems (bug #41658)

* libgui/graphics/QtHandlesUtils.cc: Rename from Utils.cc.
* libgui/graphics/QtHandlesUtils.h: Rename from Utils.h.
Change all uses.
* libgui/graphics/module.mk: Update lists.

diff --git a/libgui/graphics/BaseControl.cc b/libgui/graphics/BaseControl.cc
--- a/libgui/graphics/BaseControl.cc
+++ b/libgui/graphics/BaseControl.cc
@@ -26,17 +26,17 @@ along with Octave; see the file COPYING.
 
 #include <QEvent>
 #include <QKeyEvent>
 #include <QMouseEvent>
 #include <QWidget>
 
 #include "BaseControl.h"
 #include "ContextMenu.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 namespace QtHandles
 {
 
 static void updatePalette (const uicontrol::properties& props, QWidget* w)
 {
   QPalette p = w->palette ();
 
diff --git a/libgui/graphics/ButtonControl.cc b/libgui/graphics/ButtonControl.cc
--- a/libgui/graphics/ButtonControl.cc
+++ b/libgui/graphics/ButtonControl.cc
@@ -23,17 +23,17 @@ along with Octave; see the file COPYING.
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
 
 #include <QAbstractButton>
 
 #include "ButtonControl.h"
 #include "Container.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 namespace QtHandles
 {
 
 ButtonControl::ButtonControl (const graphics_object& go, QAbstractButton* btn)
     : BaseControl (go, btn), m_blockCallback (false)
 {
   uicontrol::properties& up = properties<uicontrol> ();
diff --git a/libgui/graphics/Canvas.cc b/libgui/graphics/Canvas.cc
--- a/libgui/graphics/Canvas.cc
+++ b/libgui/graphics/Canvas.cc
@@ -28,17 +28,17 @@ along with Octave; see the file COPYING.
 #include <QList>
 #include <QMouseEvent>
 #include <QRectF>
 
 #include "Backend.h"
 #include "Canvas.h"
 #include "ContextMenu.h"
 #include "GLCanvas.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 namespace QtHandles
 {
 
 void Canvas::redraw (bool sync)
 {
   if (sync)
     qWidget ()->repaint ();
diff --git a/libgui/graphics/Container.cc b/libgui/graphics/Container.cc
--- a/libgui/graphics/Container.cc
+++ b/libgui/graphics/Container.cc
@@ -26,17 +26,17 @@ along with Octave; see the file COPYING.
 
 #include <QVBoxLayout>
 
 #include "graphics.h"
 
 #include "Canvas.h"
 #include "Container.h"
 #include "Object.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 namespace QtHandles
 {
 
 Container::Container (QWidget* parent)
   : ContainerBase (parent), m_canvas (0)
 {
   setFocusPolicy (Qt::ClickFocus);
diff --git a/libgui/graphics/ContextMenu.cc b/libgui/graphics/ContextMenu.cc
--- a/libgui/graphics/ContextMenu.cc
+++ b/libgui/graphics/ContextMenu.cc
@@ -23,17 +23,17 @@ along with Octave; see the file COPYING.
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
 
 #include <QMenu>
 
 #include "Backend.h"
 #include "ContextMenu.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 namespace QtHandles
 {
 
 ContextMenu* ContextMenu::create (const graphics_object& go)
 {
   Object* parent = Object::parentObject (go);
 
diff --git a/libgui/graphics/EditControl.cc b/libgui/graphics/EditControl.cc
--- a/libgui/graphics/EditControl.cc
+++ b/libgui/graphics/EditControl.cc
@@ -24,17 +24,17 @@ along with Octave; see the file COPYING.
 #include <config.h>
 #endif
 
 #include <QLineEdit>
 
 #include "Container.h"
 #include "EditControl.h"
 #include "TextEdit.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 namespace QtHandles
 {
 
 EditControl* EditControl::create (const graphics_object& go)
 {
   Object* parent = Object::parentObject (go);
 
diff --git a/libgui/graphics/Figure.cc b/libgui/graphics/Figure.cc
--- a/libgui/graphics/Figure.cc
+++ b/libgui/graphics/Figure.cc
@@ -38,17 +38,17 @@ along with Octave; see the file COPYING.
 #include <QTimer>
 #include <QToolBar>
 
 #include "Canvas.h"
 #include "Container.h"
 #include "Figure.h"
 #include "FigureWindow.h"
 #include "MouseModeActionGroup.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 namespace QtHandles
 {
 
 #define ABOUT_TEXT "<b>QtHandles</b> - a Qt-based toolkit for <a href=\"http://www.octave.org\">Octave</a>.<br><br>Copyright (C) 2011-2014 Michael Goffioul"
 
 DECLARE_GENERICEVENTNOTIFY_SENDER(MenuBar, QMenuBar);
 
diff --git a/libgui/graphics/ListBoxControl.cc b/libgui/graphics/ListBoxControl.cc
--- a/libgui/graphics/ListBoxControl.cc
+++ b/libgui/graphics/ListBoxControl.cc
@@ -23,17 +23,17 @@ along with Octave; see the file COPYING.
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
 
 #include <QListWidget>
 
 #include "Container.h"
 #include "ListBoxControl.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 namespace QtHandles
 {
 
 static void updateSelection (QListWidget* list, const Matrix& value)
 {
   octave_idx_type n = value.numel ();
   int lc = list->count ();
diff --git a/libgui/graphics/Menu.cc b/libgui/graphics/Menu.cc
--- a/libgui/graphics/Menu.cc
+++ b/libgui/graphics/Menu.cc
@@ -26,17 +26,17 @@ along with Octave; see the file COPYING.
 
 #include <QAction>
 #include <QMainWindow>
 #include <QMenu>
 #include <QMenuBar>
 
 #include "Figure.h"
 #include "Menu.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 namespace QtHandles
 {
 
 static QKeySequence accelSequence (const uimenu::properties& up)
 {
   std::string s (up.get_accelerator ());
 
diff --git a/libgui/graphics/Object.cc b/libgui/graphics/Object.cc
--- a/libgui/graphics/Object.cc
+++ b/libgui/graphics/Object.cc
@@ -23,17 +23,17 @@ along with Octave; see the file COPYING.
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
 
 #include <QVariant>
 
 #include "Backend.h"
 #include "Object.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 namespace QtHandles
 {
 
 Object::Object (const graphics_object& go, QObject* obj)
   : QObject (), m_handle (go.get_handle ()), m_qobject (0)
 {
   gh_manager::auto_lock lock (false);
diff --git a/libgui/graphics/ObjectFactory.cc b/libgui/graphics/ObjectFactory.cc
--- a/libgui/graphics/ObjectFactory.cc
+++ b/libgui/graphics/ObjectFactory.cc
@@ -44,17 +44,17 @@ along with Octave; see the file COPYING.
 #include "PushButtonControl.h"
 #include "PushTool.h"
 #include "RadioButtonControl.h"
 #include "SliderControl.h"
 #include "TextControl.h"
 #include "ToggleButtonControl.h"
 #include "ToggleTool.h"
 #include "ToolBar.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 namespace QtHandles
 {
 
 ObjectFactory* ObjectFactory::instance (void)
 {
   static ObjectFactory s_instance;
   static bool s_instanceCreated = false;
diff --git a/libgui/graphics/Panel.cc b/libgui/graphics/Panel.cc
--- a/libgui/graphics/Panel.cc
+++ b/libgui/graphics/Panel.cc
@@ -29,17 +29,17 @@ along with Octave; see the file COPYING.
 #include <QLabel>
 #include <QMouseEvent>
 #include <QTimer>
 
 #include "Canvas.h"
 #include "Container.h"
 #include "ContextMenu.h"
 #include "Panel.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 namespace QtHandles
 {
 
 static int frameStyleFromProperties (const uipanel::properties& pp)
 {
   if (pp.bordertype_is ("none"))
     return QFrame::NoFrame;
diff --git a/libgui/graphics/PopupMenuControl.cc b/libgui/graphics/PopupMenuControl.cc
--- a/libgui/graphics/PopupMenuControl.cc
+++ b/libgui/graphics/PopupMenuControl.cc
@@ -23,17 +23,17 @@ along with Octave; see the file COPYING.
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
 
 #include <QComboBox>
 
 #include "Container.h"
 #include "PopupMenuControl.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 namespace QtHandles
 {
 
 PopupMenuControl* PopupMenuControl::create (const graphics_object& go)
 {
   Object* parent = Object::parentObject (go);
 
diff --git a/libgui/graphics/PushButtonControl.cc b/libgui/graphics/PushButtonControl.cc
--- a/libgui/graphics/PushButtonControl.cc
+++ b/libgui/graphics/PushButtonControl.cc
@@ -24,17 +24,17 @@ along with Octave; see the file COPYING.
 #include <config.h>
 #endif
 
 #include <QPushButton>
 #include <QtDebug>
 
 #include "PushButtonControl.h"
 #include "Container.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 namespace QtHandles
 {
 
 PushButtonControl* PushButtonControl::create (const graphics_object& go)
 {
   Object* parent = Object::parentObject (go);
 
diff --git a/libgui/graphics/Utils.cc b/libgui/graphics/QtHandlesUtils.cc
rename from libgui/graphics/Utils.cc
rename to libgui/graphics/QtHandlesUtils.cc
--- a/libgui/graphics/Utils.cc
+++ b/libgui/graphics/QtHandlesUtils.cc
@@ -32,17 +32,17 @@ along with Octave; see the file COPYING.
 
 #include "ov.h"
 #include "graphics.h"
 
 #include "Backend.h"
 #include "Container.h"
 #include "KeyMap.h"
 #include "Object.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 namespace QtHandles
 {
 
 namespace Utils
 {
 
 QString fromStdString (const std::string& s)
diff --git a/libgui/graphics/Utils.h b/libgui/graphics/QtHandlesUtils.h
rename from libgui/graphics/Utils.h
rename to libgui/graphics/QtHandlesUtils.h
diff --git a/libgui/graphics/RadioButtonControl.cc b/libgui/graphics/RadioButtonControl.cc
--- a/libgui/graphics/RadioButtonControl.cc
+++ b/libgui/graphics/RadioButtonControl.cc
@@ -23,17 +23,17 @@ along with Octave; see the file COPYING.
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
 
 #include <QRadioButton>
 
 #include "RadioButtonControl.h"
 #include "Container.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 namespace QtHandles
 {
 
 RadioButtonControl* RadioButtonControl::create (const graphics_object& go)
 {
   Object* parent = Object::parentObject (go);
 
diff --git a/libgui/graphics/SliderControl.cc b/libgui/graphics/SliderControl.cc
--- a/libgui/graphics/SliderControl.cc
+++ b/libgui/graphics/SliderControl.cc
@@ -23,17 +23,17 @@ along with Octave; see the file COPYING.
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
 
 #include <QScrollBar>
 
 #include "Container.h"
 #include "SliderControl.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 #define RANGE_INT_MAX 1000000
 
 namespace QtHandles
 {
 
 SliderControl* SliderControl::create (const graphics_object& go)
 {
diff --git a/libgui/graphics/TextControl.cc b/libgui/graphics/TextControl.cc
--- a/libgui/graphics/TextControl.cc
+++ b/libgui/graphics/TextControl.cc
@@ -23,17 +23,17 @@ along with Octave; see the file COPYING.
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
 
 #include <QLabel>
 
 #include "Container.h"
 #include "TextControl.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 namespace QtHandles
 {
 
 TextControl* TextControl::create (const graphics_object& go)
 {
   Object* parent = Object::parentObject (go);
 
diff --git a/libgui/graphics/ToggleButtonControl.cc b/libgui/graphics/ToggleButtonControl.cc
--- a/libgui/graphics/ToggleButtonControl.cc
+++ b/libgui/graphics/ToggleButtonControl.cc
@@ -23,17 +23,17 @@ along with Octave; see the file COPYING.
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
 
 #include <QPushButton>
 
 #include "ToggleButtonControl.h"
 #include "Container.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 namespace QtHandles
 {
 
 ToggleButtonControl* ToggleButtonControl::create (const graphics_object& go)
 {
   Object* parent = Object::parentObject (go);
 
diff --git a/libgui/graphics/ToolBar.cc b/libgui/graphics/ToolBar.cc
--- a/libgui/graphics/ToolBar.cc
+++ b/libgui/graphics/ToolBar.cc
@@ -31,17 +31,17 @@ along with Octave; see the file COPYING.
 #include <QIcon>
 #include <QMainWindow>
 #include <QPixmap>
 #include <QTimer>
 #include <QToolBar>
 
 #include "Figure.h"
 #include "ToolBar.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 namespace QtHandles
 {
 
 static QAction* addEmptyAction (QToolBar* bar)
 {
   static QIcon _empty;
 
diff --git a/libgui/graphics/ToolBarButton.cc b/libgui/graphics/ToolBarButton.cc
--- a/libgui/graphics/ToolBarButton.cc
+++ b/libgui/graphics/ToolBarButton.cc
@@ -23,17 +23,17 @@ along with Octave; see the file COPYING.
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
 
 #include <QAction>
 #include <QWidget>
 
 #include "ToolBarButton.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 namespace QtHandles
 {
 
 template <class T>
 ToolBarButton<T>::ToolBarButton (const graphics_object& go, QAction* action)
     : Object (go, action), m_separator (0)
 {
diff --git a/libgui/graphics/__init_qt__.cc b/libgui/graphics/__init_qt__.cc
--- a/libgui/graphics/__init_qt__.cc
+++ b/libgui/graphics/__init_qt__.cc
@@ -31,17 +31,17 @@ along with Octave; see the file COPYING.
 #include <QPalette>
 #include <QRegExp>
 
 #include "graphics.h"
 #include "toplev.h"
 #include "defun.h"
 
 #include "Backend.h"
-#include "Utils.h"
+#include "QtHandlesUtils.h"
 
 namespace QtHandles
 {
 
 static bool qtHandlesInitialized = false;
 
 bool __init__ (void)
 {
diff --git a/libgui/graphics/module.mk b/libgui/graphics/module.mk
--- a/libgui/graphics/module.mk
+++ b/libgui/graphics/module.mk
@@ -51,25 +51,25 @@ noinst_HEADERS += \
   graphics/MouseModeActionGroup.h \
   graphics/Object.h \
   graphics/ObjectFactory.h \
   graphics/ObjectProxy.h \
   graphics/Panel.h \
   graphics/PopupMenuControl.h \
   graphics/PushButtonControl.h \
   graphics/PushTool.h \
+  graphics/QtHandlesUtils.h \
   graphics/RadioButtonControl.h \
   graphics/SliderControl.h \
   graphics/TextControl.h \
   graphics/TextEdit.h \
   graphics/ToggleButtonControl.h \
   graphics/ToggleTool.h \
   graphics/ToolBar.h \
   graphics/ToolBarButton.h \
-  graphics/Utils.h \
   graphics/gl-select.h
 
 graphics_libgui_graphics_la_SOURCES = \
   graphics/__init_qt__.cc \
   graphics/Backend.cc \
   graphics/BaseControl.cc \
   graphics/ButtonControl.cc \
   graphics/Canvas.cc \
@@ -87,25 +87,25 @@ graphics_libgui_graphics_la_SOURCES = \
   graphics/MouseModeActionGroup.cc \
   graphics/Object.cc \
   graphics/ObjectFactory.cc \
   graphics/ObjectProxy.cc \
   graphics/Panel.cc \
   graphics/PopupMenuControl.cc \
   graphics/PushButtonControl.cc \
   graphics/PushTool.cc \
+  graphics/QtHandlesUtils.cc \
   graphics/RadioButtonControl.cc \
   graphics/SliderControl.cc \
   graphics/TextControl.cc \
   graphics/TextEdit.cc \
   graphics/ToggleButtonControl.cc \
   graphics/ToggleTool.cc \
   graphics/ToolBar.cc \
   graphics/ToolBarButton.cc \
-  graphics/Utils.cc \
   graphics/gl-select.cc
 
 nodist_graphics_libgui_graphics_la_SOURCES = $(octave_gui_graphics_MOC) $(octave_gui_graphics_RC)
 
 graphics_libgui_graphics_la_CPPFLAGS = \
   $(AM_CPPFLAGS) \
   $(FT2_CFLAGS) \
   $(FONTCONFIG_CPPFLAGS) \
